plugins/inputs/knx_listener/knx_helpers.go

```diff
@@ -0,0 +1,205 @@
+package knx_listener
+
+import (
+	"fmt"
+	"reflect"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/vapourismo/knx-go/knx"
+	"github.com/vapourismo/knx-go/knx/cemi"
+	"github.com/vapourismo/knx-go/knx/dpt"
+)
+
+// Helper to abstract away router or tunnel interfaces
+type KNXInterface interface {
+	Inbound() <-chan knx.GroupEvent
+	Close()
+}
+
+func GetKNXInterface(mode string, address string) (client KNXInterface, err error) {
```

Comment:
it's fine to name return values, but typically you should still explicitly return the values rather than implicitly setting them and letting the function return later.

Reply:
Done